models/data_maps.go

```diff
@@ -6,16 +6,22 @@ import (
 	"encoding/json"
 	"math"
 	"time"
-
 	"github.com/gobuffalo/pop"
 	"github.com/gobuffalo/validate"
 	"github.com/gobuffalo/validate/validators"
 )
 
-var ChunkStatus map[string]int
-
 const fileBytesChunkSize = float64(2817)
 
+const (
+	Pending int = iota + 1
+	Unassigned
+	Unverified
+	Complete
+	Confirmed
+	Error
```

Comment:
Is it possible to make `Error` -1?

Reply:
Sure.  So far we haven't used error, and I'm not sure we even need it.  But I will change it.